src/org/jitsi/impl/neomedia/MediaStreamStatsImpl.java

```diff
@@ -1209,13 +1209,12 @@ private void setRttMs(long rttMs)
             {
                 // RemoteBitrateEstimator is a CallStatsObserver and
                 // VideoMediaStream has a RemoteBitrateEstimator.
-                MediaStream mediaStream = this.mediaStreamImpl;
+                MediaStreamImpl mediaStream = this.mediaStreamImpl;
```

Comment:
I removed the getter because 1st it was possible to do (only implementation classes within libjitsi use it) and 2nd it leads to a _lighter_ `MediaStream` interface.